lighthouse/vars.env

```diff
@@ -33,7 +33,7 @@ GENESIS_DELAY=0
 BOOTNODE_PORT=4242
 
 # Network ID and Chain ID of local eth1 test network
-CHAIN_ID=4242
+CHAIN_ID=1
```

Comment:
Comment above says this should be chain-id for the "local eth1 test network", so why change it to mainnet?

Reply:
prysm / geth currently is set to 1, and not changing it causes test errors